quickstarts/skykit/config.yml

```
@@ -0,0 +1,44 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: skykit
+
+# Displayed in the UI (required)
+title: Skykit Dashboards
+
+# Long-form description of the quickstart (required)
+description: |
+  With our secure, cloud-based dashboard tool, you'll be able to showcase your visual dashboards to 1, 5, 100 or more screens throughout your organization to drive smart decision making and employee engagement. Skykit allows you to securely share your dashboards and build alignment and engagement across your whole team not just the data experts.
+
+  For more information on how New Relic customers use Skykit visit [Skykit New Relic Dashboards](https://www.skykit.com/products/dashboards/connections/new-relic/).
+
+  Skykit sales: sales@skykit.com
+
+  Skykit support: support@skykit.com 
+
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Display New Relic dashboards anywhere for the entire company to see.
+
+  
+# Support level: New Relic | Verified | Community (required)
+level: Verified 
+
+# Authors of the quickstart (required)
+authors:
+  - Skykit
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - cms
+  - content management system 
+  - newrelic partner
+
+# Documentation references
+documentation:
+  - name: Get Started With Skykit Dashboards 
+    url: https://www.skykit.com/products/dashboards/connections/new-relic/
+    description: Connect with Skykit's team to get started displaying your New Relic dashboards 
+
+# Content / Design
+icon: logo.png
+website: https://www.skykit.com/products/dashboards/connections/new-relic/
```

```suggestion
website: https://www.skykit.com
```

This should just be the company site, you can leave the documentation link in the doc references.